ports/stm32/boards/PYBD_SF6/mpconfigboard.h

```diff
@@ -86,28 +86,28 @@
 
 #define MICROPY_BOARD_EARLY_INIT    board_early_init_sf6
 
-#define CHIP_PARAMS0 (spi_bdev.spiflash.chip_params)
-#define CHIP_PARAMS1 (spi_bdev2.spiflash.chip_params)
+#define MICROPY_BOARD_SPIFLASH_CHIP_PARAMS0 (spi_bdev.spiflash.chip_params)
+#define MICROPY_BOARD_SPIFLASH_CHIP_PARAMS1 (spi_bdev2.spiflash.chip_params)
```

Comment:
Minor, but suggest a couple of inline comments here about `// QSPI Flash 1`, `// QSPI Flash 2` or similar, to link these back to the hardware naming scheme